target/linux/ramips/dts/mt7621_ubiquiti_edgerouterx.dtsi

```diff
@@ -36,41 +36,44 @@
 &nand {
 	status = "okay";
 
-	partitions {
-		compatible = "fixed-partitions";
-		#address-cells = <1>;
-		#size-cells = <1>;
-
-		partition@0 {
-			label = "u-boot";
-			reg = <0x0 0x80000>;
-			read-only;
-		};
+	nand@0 {
+		reg = <0>;
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
 
-		partition@80000 {
-			label = "u-boot-env";
-			reg = <0x80000 0x60000>;
-			read-only;
-		};
+			partition@0 {
```

Comment:
Some whitespace issues?

Reply:
Isn't this shift a necessary consequence of the additional nand-chip level?

Reply:
Ahh, good point, missed that, thanks!